sdk/core/src/test/java/org/apache/mesos/scheduler/plan/DefaultPlanManagerTest.java

```diff
@@ -208,6 +205,95 @@ public void testUpdate() {
         verify(mockBlock, times(1)).update(any());
     }
 
+    @Test
+    public void testAllDirtyAssets() {
+        when(reconciler.isReconciled()).thenReturn(false);
+        final TestBlock block1 = new TestBlock("test-block-1");
+        final TestBlock block2 = new TestBlock("test-block-2");
+
+        final DefaultPhase phase = new DefaultPhase(
+                "phase-1",
+                Arrays.asList(block1, block2),
+                new SerialStrategy<>(),
+                Collections.emptyList());
+
+        DefaultPlan waitingPlan = new DefaultPlan(
+                "test-plan",
+                Arrays.asList(phase),
+                new SerialStrategy<>(),
+                Collections.emptyList()
+        );
```

Comment:
inconsistent style.  Generally we leave that last `);` on the previous line as on line 218